contracts/KYCRecordRegistry2.sol

```diff
@@ -0,0 +1,163 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Based on code from MACI (https://github.com/appliedzkp/maci/blob/7f36a915244a6e8f98bacfe255f8bd44193e7919/contracts/sol/IncrementalMerkleTree.sol)
+pragma solidity ^0.8.7;
+pragma abicoder v2;
+
+// OpenZeppelin v4
+import { Initializable } from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+import { SNARK_SCALAR_FIELD } from "./helpers/Globals.sol";
+
+import { PoseidonT3 } from "./helpers/Poseidon.sol";
+
+import { KYCCenterRegistry } from "./KYCCenterRegistry.sol";
+
+/**
+ * @title KYCRecordRegistry
+ * @author Galactica dev team
+ * @notice Batch Incremental Merkle Tree for zkKYC records
+ * Relevant external contract calls should be in those functions, not here
+ */
+contract KYCRecordRegistry2 is Initializable {
```

Comment:
Please use descriptive names for contracts and files.
For me it is confusing what  `KYCRecordRegistry2` is for.

In my opinion it is better to add improvements directly to the original file or name the variant in a way that makes it obvious what is different and what it is needed for.
Doing it in the same file has the advantage that improvements apply to all future versions of it.

Or is this just an accidental commit of an unused file?